source/linear_elasticity.cpp

```diff
@@ -372,8 +372,16 @@ namespace Solid
     auto cell = dof_handler.begin_active();
     auto scalar_cell = scalar_dof_handler.begin_active();
     std::vector<types::global_dof_index> dof_indices(scalar_fe.dofs_per_cell);
+
+    // containers for cell-wise stress output
+    cellwise_sxx.reinit(triangulation.n_active_cells());
```

Comment:
Should be compatible with 3D, i.e.,  when ``dim==3`` there is z direction.

Reply:
Fixed